Assets/Plugins/Source/Core/EOSManager.cs

```diff
@@ -323,64 +323,50 @@ static void print(string toPrint)
             }
 
             //-------------------------------------------------------------------------
-            public void AddConnectLoginListener(IEOSOnConnectLogin connectLogin)
+            public void AddConnectLoginListener(IConnectInterfaceEventListener connectLogin)
             {
                 s_onConnectLoginCallbacks.Add(connectLogin.OnConnectLogin);
             }
-
-            public void AddAuthLoginListener(IEOSOnAuthLogin authLogin)
+            public void RemoveConnectLoginListener(IConnectInterfaceEventListener connectLogin)
             {
-                s_onAuthLoginCallbacks.Add(authLogin.OnAuthLogin);
-            }
-
-            public void AddAuthLogoutListener(IEOSOnAuthLogout authLogout)
-            {
-                s_onAuthLogoutCallbacks.Add(authLogout.OnAuthLogout);
+                s_onConnectLoginCallbacks.Remove(connectLogin.OnConnectLogin);
             }
 
-            public void AddApplicationCloseListener(Action listener)
+            public void AddAuthInterfaceEventListener(IAuthInterfaceEventListener listener)
```

Comment:
Likewise for this, it is also changing the API parameters when it shouldn't.

Reply:
Fixed :)